torch/onnx/symbolic_opset9.py

```diff
@@ -835,24 +835,38 @@ def _prepare_onnx_paddings(dim, pad):
     paddings = paddings[-2::-2] + paddings[-1::-2]
     return paddings
 
+def _convert_padding_node(padding):
+    padding = sym_help._maybe_get_const(padding, 'is')
+    if sym_help._is_value(padding) and sym_help._is_packed_list(padding):
+        input_list = sym_help._unpack_list(padding)
+        input_is_constant = [v.node().kind() == 'onnx::Constant' for v in input_list]
+        if not all(input_is_constant):
+            return sym_help._onnx_opset_unsupported_detailed('Pad', 9, 11, 'The sizes of the padding must be constant')
+        padding = [int(v.node()['value']) for v in padding.node().inputs()]
+    return padding
 
-@parse_args('v', 'is', 'f')
 def constant_pad_nd(g, input, padding, value):
     mode = "constant"
+    try:
```

Comment:
This looks okay for now. If see this type of error handling (per opset) repeating more often, we should handle it in a separate function to make it more generic.